### ISSUE TITLE
Add support for new UIMA JSON serialization

### ISSUE THREAD

**Issue by reckart** [dkpro/dkpro-core#455]
```
Add support for new UIMA JSON serialization

https://issues.apache.org/jira/browse/UIMA-3969
```

Original issue reported on code.google.com by `richard.eckart` on 2014-08-15 13:22:19

**Comment by reckart** [comment]
Test case is presently failing because the documentUri and documentBaseUri contain a full path which is machine specific. These need to be filtered out somehow or be ignored during comparison.

**Comment by reckart** [comment]
Failing test resolved via #710.
